pkg/modulereader/resreader.go

```diff
@@ -135,6 +135,9 @@ func GetModuleInfo(source string, kind string) (ModuleInfo, error) {
 	switch {
 	case sourcereader.IsEmbeddedPath(source) || sourcereader.IsLocalPath(source):
 		modPath = source
+		if sourcereader.IsLocalPath(source) && sourcereader.LocalModuleIsEmbedded(source) {
+			return ModuleInfo{}, fmt.Errorf("using embedded modules with local paths is no longer supported; remove the local path reference and rebuild the gcluster binary")
```

Comment:
I would either remove " and rebuild the gcluster binary" or added " and rebuild the gcluster binary if you have made any changes to it"

Reply:
Took spirit of suggestion and modified with clearer direct guidance.